main/entity/src/main/java/at/wrk/coceso/form/GroupsForm.java

```diff
@@ -1,26 +1,24 @@
 package at.wrk.coceso.form;
 
 import at.wrk.coceso.entity.Unit;
+
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class GroupsForm {
 
-  private List<Group> groups;
-
-  public GroupsForm() {
-  }
-
-  public GroupsForm(List<Unit> units) {
-    groups = units.stream().map(Group::new).collect(Collectors.toList());
-  }
+    private final List<Group> groups;
 
-  public List<Group> getGroups() {
-    return groups;
-  }
+    public GroupsForm() {
+        groups = new ArrayList<>();
```

Comment:
The list is effectively read-only, so use java.util.Collections.emptyList() instead?
I think this constructor is unnecessary anyway because the form is never created through Jackson etc afaik.

Reply:
JSP edits it and sends it back, therefore I think it is also used on the POST method.

On the long run we should make Java 11+ required at some point, then there are a lot of nice List and Set helper methods, especially for unmodifiable collections.